### PR TITLE
Created broker list option

### PR DESCRIPTION
**Pull request by Scapogo**
Small change based on issue [#48](https://github.com/zipline-live/zipline/issues/48)

### CODE REVIEW

**Comment by tibkiss** [comment]
Thanks for this PR. Unfortunately it has merge conflict due to a recent change in run.
Could you please resolve that?

Additionally, we should future proof this PR by building the list of available brokers dynamically. pkg_util.iter_module() is able to provide that information relatively easily.

CC: @bartosh

**Comment by Scapogo** [comment]
Hello,

 thanks for your comment, i will resolve it and do what you are asking. Just please give me time till Monday.

**Comment by tibkiss** [comment]
Sure, thanks for your contribution!

File: zipline/__main__.py [hunk 1]
```diff
@@ -210,6 +213,11 @@ def _(*args, **kwargs):
     metavar='DIRNAME',
     help='Directory where the realtime collected minutely bars are saved'
 )
+@click.option(
+    '--broker-list',
```
Comment (tibkiss) [inline zipline/__main__.py]: Could you please rename this option (and the variables downstream) to '--list-brokers' ? I think it shows better the intent.

File: zipline/__main__.py [hunk 2]
```diff
@@ -227,9 +235,18 @@ def run(ctx,
         broker,
         broker_uri,
         state_file,
+        broker_list,
```
Comment (tibkiss) [inline zipline/__main__.py]: Please move this variable after the `realtime_bar_target` to be in sync with the parameter order above.

Comment (tibkiss) [reply zipline/__main__.py]: Also, please rename this variable to list_brokers.

File: zipline/__main__.py [hunk 3]
```diff
@@ -227,9 +235,18 @@ def run(ctx,
...
         realtime_bar_target):
     """Run a backtest for the given algorithm.
     """
+
+    if broker_list:
+        click.echo("Supported brokers:")
+        for module_loader, name, ispkg in pkgutil.iter_modules(brokers.__path__):
```
Comment (tibkiss) [inline zipline/__main__.py]: `module_loader` and `ispkg` variables seems to be unused.
You can avoid them with: `for _, name, _ in pkgutil.iter_modules(brokers.__path__)`

File: zipline/__main__.py [hunk 4]
```diff
@@ -227,9 +235,18 @@ def run(ctx,
...
+            if name != 'broker':
+                print name
```
Comment (tibkiss) [inline zipline/__main__.py]: This line breaks python 3 support (see travis log with python 3 runs).
I'd suggest using `click.echo()`, which is consistent with the rest of this file

File: zipline/__main__.py (hunk 3 above)
Comment (tibkiss) [reply zipline/__main__.py]: Just noticed that this line is too long. Zipline runs flake8 with 79 line width allowed:
https://travis-ci.org/zipline-live/zipline/jobs/285172438#L1206

**Comment by tibkiss** [comment]
Merged into master, thanks for the contribution @Scapogo !

**Pull request merged**